README.md

```diff
@@ -332,73 +354,73 @@ sudo ./check-production.sh
 	crw-rw---- 1 root sgx     10, 124 Apr  3 21:14 /dev/sgx_vepc
 	```
 
-3. Verify the QGS service is running properly:
+3. Verify the QGS service is running properly.
 	```bash
 	sudo systemctl status qgsd
 	```
 
-4. Verify the PCCS service is running properly:
+4. Verify the PCCS service is running properly.
 	```bash
 	sudo systemctl status pccs
 	```
 
-5. To setup the PCCS in the next step, you need a subscription key for the [Intel PCS](https://api.portal.trustedservices.intel.com/provisioning-certification).
-	* If you did not request such a subscription key before, [subscribe](https://api.portal.trustedservices.intel.com/products#product=liv-intel-software-guard-extensions-provisioning-certification-service) to Intel PCS, which requires to log in (or create an account).
-	Two subscription keys are generated (for key rotation) and both can be used for the following step.
-	* If you did request such a subscription key before, [retrieve](https://api.portal.trustedservices.intel.com/manage-subscriptions) one of your keys, which requires to log in.
-	You have two subscription keys (for key rotation) and both can be used for the following step.
+5. To setup the PCCS in the next step, you need a subscription to the [Intel PCS](https://api.portal.trustedservices.intel.com/provisioning-certification) service.
+   1. [Subscribe](https://api.portal.trustedservices.intel.com/products#product=liv-intel-software-guard-extensions-provisioning-certification-service) to Intel PCS.
+      For key rotation purpose, two subscription keys are generated.
+   2. [Retrieve](https://api.portal.trustedservices.intel.com/manage-subscriptions) one of your keys and use it in the next step.
 
-6. Configure the PCCS service:
+6. Configure the PCCS service.
 
 	```bash
 	sudo /usr/bin/pccs-configure
 	```
 
-	An example configuration you can use:
+	An example configuration:
 
 	```
 	Checking nodejs version ...
 	nodejs is installed, continue...
 	Checking cracklib-runtime ...
 	Set HTTPS listening port [8081] (1024-65535) :
 	Set the PCCS service to accept local connections only? [Y] (Y/N) :
-	Set your Intel PCS API key (Press ENTER to skip) : <Enter your Intel PCS subscription key here>
-	You didn't set Intel PCS API key. You can set it later in config/default.json.
+	Set your Intel PCS API key (Press ENTER to skip) : b1b98494660fa0de93638c531d2c0df7
```

Comment:
I dont think that we should put a valid API key here, each user will have his own key

Reply:
this is an invalid sample API key just for demo purpose.